src/imap/model.rs

```diff
@@ -81,7 +151,12 @@ impl<'a> ImapConnector<'a> {
         Ok(())
     }
 
-    pub fn remove_flags(&mut self, mbox: &str, uid_seq: &str, flags: &str) -> Result<()> {
+    /// Remove the flags to the message by the given information. Take a look on the example above.
+    /// It's pretty similar.
+    pub fn remove_flags(&mut self, mbox: &str, uid_seq: &str, flags: Flags) -> Result<()> {
+
```

Comment:
Why adding a new line here? It does not really fit the global style of the code base.